pom.xml

```diff
@@ -61,7 +61,7 @@
     <derby.version>10.10.2.0</derby.version>
     <beeju.version>1.2.1</beeju.version>
     <hamcrest-library.version>1.3</hamcrest-library.version>
-    <housekeeping.version>2.0.0</housekeeping.version>
+    <housekeeping.version>2.1.1-SNAPSHOT</housekeeping.version>
```

Comment:
I suggest we bump to 2.2.0 release for HK?

Reply:
Maybe even 3.0.0, see my comments over there.

Reply:
ack lets go for 3.0.0